src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,61 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String DELIMITER = ",";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        int[] totals = readData(fromFileName);
+        int supplyTotal = totals[0];
```

Comment:
It's better to create method `String generateReport(int[] totals)` and replace logic there
```
int supplyTotal = totals[0];
int buyTotal = totals[1];
int result = supplyTotal - buyTotal;
StringBuilder report = new StringBuilder();
report.append(SUPPLY).append(DELIMITER).append(supplyTotal).append(System.lineSeparator()).append(BUY)......

```

and provide report to method writeData